docs/users.md

```diff
@@ -31,3 +31,4 @@
 [Trioxis](https://www.trioxis.com/)
 
 [Real World React](https://realworldreact.com) ([source](https://github.com/realworldreact/realworldreact-website))
+[Bootstrp JSS](https://sammi.github.io/bootstrap-jss)([source](https://github.com/sammi/bootstrap-jss))
```

Comment:
Typo, should be `Bootstrap`

Reply:
yes, there is a typo. I am thinking that we can place this lib and mui together on the first readme section, so that it is easier to discover, but I think this lib needs better storybook integration, currently there is only a button. As soon as all components are in storybook.

Reply:
Thanks guys, working on more stories.